src/timestamp-oracle/src/lib.rs

```diff
@@ -7,6 +7,74 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+// BEGIN LINT CONFIG
```

Comment:
Just a heads up that this might conflict with https://github.com/MaterializeInc/materialize/pull/23725, depending on which lands first. Definitely no need to wait for that PR to merge though, I can rebase on `main`!

Reply:
Yup, I was about to give you the same heads up! This is set to merge on green, so hopefully it lands imminently 🤞

Reply:
Just merged the PR, looks like you rebased on `main`, hopefully it wasn't too bad 🙈